### ISSUE TITLE
[Wallet] Control allowed gas price and pending transactions due to exceeded gas limit

### ISSUE THREAD

**Issue by Jihed525** [Meeds-io/meeds#344]
**Is your feature request related to a problem? Please describe.**
Gas price could be rocketing unexpected high levels which makes buying a pen cost a fortune in transactions, this why a maximum amount is set. 
yet, setting a maximum without being able to edit it could block some needed transactions though.

**Describe the solution you'd like**
Allow Wallet administrator to manage the maximum allowed gas price via UI.
display a list of pending transactions and allow administrator to whether delete or confirm each one, select some or all.

**Comment by Jihed525** [comment]
just edited the description by mentioning the UI item

**Comment by plamarque** [comment]
yes, admin may be able to agree to pay market price for gas. But this possibility could also be offered to the user to boost execution manually. The UI should give him an estimate of the gas cost for the transaction at market price.
